nibabel/parrec.py

```diff
@@ -610,6 +610,7 @@ def __getitem__(self, slicer):
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
         fake_data = strided_scalar(self._shape)
+        fake_data.flags.writeable = False
```

Comment:
I can try, but I really don't think the mutability of the shape tuple is relevant. 

On October 15, 2015 12:35:57 AM EDT, Matthew Brett <notifications@github.com> wrote:
>> @@ -610,6 +610,7 @@ def __getitem__(self, slicer):
>>          # Broadcast scaling to shape of original data
>>          slopes, inters = self._slice_scaling
>>          fake_data = strided_scalar(self._shape)
>> +        fake_data.flags.writeable = False
>
>How about
>
>```
># Create broadcast array using writeable memory for numpy 1.10
>fake_data = strided_scalar(np.array(self._shape))
>```
>
>?  Only because the writeable line looks a bit frightening.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/nipy/nibabel/pull/358/files#r42082838

-- 
Chris Markiewicz